websphere-liberty/content.md

```diff
@@ -1,23 +1,41 @@
 # Overview
 
-This image contains IBM WebSphere Application Server for Developers Liberty Profile. For more information on WebSphere Application Server Liberty Profile, see the [WASdev](https://developer.ibm.com/wasdev/docs/category/getting-started/) site. Due to the current lack of geographic export controls on Docker Hub it has been necessary to remove the IBM JRE from this image. The image now builds on the Debian based `java:jre` OpenJRE official repository image. To build an Ubuntu image with the IBM JRE, use the Dockerfiles available in the [ibmjre](https://github.com/WASdev/ci.docker/tree/ibmjre/websphere-liberty/8.5.5) branch on GitHub.
+The images in this repository contain IBM WebSphere Application Server for Developers Liberty Profile. For more information on WebSphere Application Server Liberty, see the [WASdev](https://developer.ibm.com/wasdev/docs/category/getting-started/) site. Due to the current lack of geographic export controls on Docker Hub it has been necessary to remove the IBM Java Runtime Environment from these images. The images now build on the Debian based `java:jre` OpenJRE official repository image. To build an Ubuntu image with the IBM JRE, use the Dockerfiles available in the [ibmjre](https://github.com/WASdev/ci.docker/tree/ibmjre/websphere-liberty/8.5.5) branch on GitHub.
+
+# Images
+
+There are multiple images available in this repository. The image with the tag `beta` contains the contents of the runtime JAR for the latest monthly beta. The other images are all based on the latest generally available fix pack.
+
+The `kernel` image contains just the Liberty kernel and no additional runtime features. This image can be used as the basis for custom built images that contain only the features required for a specific application. For example, the following Dockerfile starts with this image, copies in the `server.xml` that lists the features required by the application, and then uses the `installUtility` command to download those features from the online repository.
+
+	FROM websphere-liberty:kernel
+	COPY server.xml /opt/ibm/wlp/usr/servers/defaultServer/
+	RUN installUtility install --acceptLicense defaultServer
```

Comment:
Might want to make this a fenced block with a language so that the Docker Hub renders it better.